sdks/java/io/hadoop-format/src/main/java/org/apache/beam/sdk/io/hadoop/format/HadoopFormatIO.java

```diff
@@ -702,6 +702,15 @@ public void populateDisplayData(DisplayData.Builder builder) {
                     "mapreduce.job.inputformat.class",
                     hadoopConfig.get("mapreduce.job.inputformat.class"))
                 .withLabel("InputFormat Class"));
+        final String inputDir = hadoopConfig.get("mapreduce.input.fileinputformat.inputdir");
+        if (inputDir != null) {
+          builder.add(
+              DisplayData.item(
+                      "mapreduce.input.fileinputformat.inputdir",
+                      String.format(
+                          "%s...", inputDir.substring(0, Math.min(inputDir.length(), 250))))
```

Comment:
How about this?

```
builder.add(
              DisplayData.item(
                      "mapreduce.input.fileinputformat.inputdir",
                      StringUtils.abbreviate(inputDir, 250)
              ));
```

ref: https://commons.apache.org/proper/commons-lang/javadocs/api-2.6/org/apache/commons/lang/StringUtils.html#abbreviate%28java.lang.String,%20int%29

Reply:
oh, perfect, that does everything I need :) updated!

Reply:
ah nice! done.